lib/std/compress/xz/block.zig

```diff
@@ -217,7 +217,7 @@ pub fn Decoder(comptime ReaderType: type) type {
                     if (status == 1)
                         try self.accum.reset(self.allocator);
 
-                    const size = try packed_reader.readIntBig(u16) + 1;
+                    const size = @as(u17, try packed_reader.readIntBig(u16)) + 1;
```

Comment:
Note that this will be superseded by/conflict with https://github.com/ziglang/zig/pull/14518#discussion_r1095228669

Let me know if it'd be better to defer this fix to that PR